server/repository/memrepo.py

```diff
@@ -42,3 +42,8 @@ def delete(self, shift_id):
                 self.data.remove(item)
                 return
 
+    def delete_shift_use_case(self, shift_id, user_id):
```

Comment:
Instead of adding this code for time being, you can just make comments about what should be done here.

Reply:
This has to be here it's the function that uses Python to remove the shift, Nhi didn't have to do it since the function she needed already is made. I don't think we should make someone reimplement it later, as it belongs here and does not impact the other back-end functionality.